src/WalterModem.h

```diff
@@ -4548,6 +4563,33 @@ class WalterModem
          * expected to be at least SPI_FLASH_SEC_SIZE = 4K
          */
         static void offlineMotaUpgrade(uint8_t *otaBuffer);
+
+        /**
+         * @brief converts a given combination of sconds,minutes, hours to a TAU approximation
+         *
+         * this function is an approximation because it uses a multiplier internally.
+         * 
+         * 
+         * @param seconds uint32_t
```

Comment:
I'm still missing parameter descriptions. Please also remove superfluous newlines and start sentences with a capital letter.